app/controllers/forget_passwords_controller.rb

```diff
@@ -0,0 +1,39 @@
+class ForgetPasswordsController < ApplicationController
+  skip_before_action :authorize
+
+  def show
+  end
+
+  def create
+    if @user = User.find_by(email: params[:email])
+      @user.update_columns(password_reset_token: SecureRandom.uuid)
+      UserMailer.reset_password(@user).deliver_later
+      message = 'Password reset mail sent to your email'
+    else
+      message = 'No such registered email exists'
+    end
+    redirect_to forget_password_path, notice: message
+  end
+
+  def reset
+    if params[:password_reset_token] && @user = User.find_by(password_reset_token: params[:password_reset_token])
+      render :reset
+    else
+      redirect_to forget_password_path, notice: 'Invalid password reset request'
+    end
+  end
+
+  def update
```

Comment:
You're updating in user entity but this is password_controller so it shouldn't be called as update